genie-web/src/test/java/com/netflix/genie/web/jpa/entities/JobEntityUnitTests.java

```diff
@@ -917,6 +917,20 @@ public void canSetV4() {
         Assert.assertTrue(this.jobEntity.isV4());
     }
 
+    /**
+     * Test setter/getter.
+     */
+    @Test
+    public void canSetRequestedArchiveLocationPrefix() {
+        Assert.assertFalse(this.jobEntity.getRequestedArchiveLocationPrefix().isPresent());
+        final String requestedArchiveLocationPrefix = UUID.randomUUID().toString();
+        this.jobEntity.setRequestedArchiveLocationPrefix(requestedArchiveLocationPrefix);
+        Assert.assertThat(
+            this.jobEntity.getRequestedArchiveLocationPrefix().orElse(UUID.randomUUID().toString()),
```

Comment:
I find the `orElse` here a bit confusing... how about just `.get`?

Reply:
It is confusing. Will fix